src/explorer.ts

```diff
@@ -333,12 +333,15 @@ class KubernetesSelectorResource extends KubernetesResource {
         if (!this.selector) {
             return [];
         }
-        const selectors = [];
-        for (const sel in this.selector.matchLabels) {
-            selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
+        let selectorString = "";
+        if (this.selector.matchLabels) {
+            const selectors = [];
+            for (const sel in this.selector.matchLabels) {
+                selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
+            }
+            selectorString = "-l" + selectors.join(",");
```

Comment:
`-l ${selectors.join(",")}`?  Seems like we'd normally put a space between an option flag and its value, though maybe it isn't formally necessary...?

Reply:
Uh there were meant to be backticks around that but I guess Markdown ated them

Reply:
fixing this.